source/trainer.py

```diff
@@ -125,17 +135,36 @@ def tokenize(data):
     train_dataset=train_data_tokenized,
     eval_dataset=validate_data_tokenized,
 )
-
+if MODEL_RUN_IN != "gdrive":  # it does not work from gdrive
+    trainer.args.push_to_hub = True
```

Comment:
nice I think this push_to_hub will make all checkpoints automatically push to hub without waiting to finalize it. but we should get rid of the line above so that it would run everytime

Reply:
see my comment above. so far this push from gdrive did not work for me. but I haven't try from the collab drive. I saw some comments reporting that it worked well from there...